pkg/publish.go

```diff
@@ -112,12 +109,17 @@ func Publish(file string, rootID string, dryRun bool) (id string, err error) {
 			return "", err
 		}
 
-		imageSrcFiles, err := getImageSrcFiles(sourceOutput, file)
+		imageSrcFiles, err := GetImageSrcFiles(sourceOutput, file)
 		if err != nil {
 			return "", err
 		}
 
-		pageContent, err := replaceImagesWithAttachments(imageSrcFiles, file, sourceOutput, c.ID, wiki, uri)
+		pageContent, err := ReplaceImagesWithAttachments(imageSrcFiles, file, sourceOutput, c.ID, wiki, uri)
+		if err != nil {
+			return "", err
+		}
+
+		pageContent, err = ReplaceRelativeLinks(file, pageContent, uri, repoBrowseURL, repoRoot)
```

Comment:
All these functions can be private since they are within the same `dox` package.

Reply:
This makes sense, I was wondering why I didn't have to import that file, but if its all same package then that helps me understand.

- [ ] make functions private